notebook/index.ts

```diff
@@ -3,6 +3,7 @@ import mermaid from 'mermaid';
 import type { RendererContext } from 'vscode-notebook-renderer';
 import { renderMermaidBlocksInElement } from '../markdownPreview/mermaid';
 import { extendMarkdownItWithMermaid } from '../src/mermaid';
+import mindmap from '@mermaid-js/mermaid-mindmap'
```

Comment:
Actually do we even need to do this if we upgrade to mermaid 9.4.0? https://mermaid.js.org/syntax/mindmap.html#integrating-with-your-library-website

Looks like it may be included by default now?

Reply:
Just make a PR updating the version as you like and I’ll close this one. We don’t need this code if it’s just included.